app/policies/book_policy.rb

```diff
@@ -0,0 +1,9 @@
+class BookPolicy < ApplicationPolicy
+  def update?
+    user == record.user # Allow updating a book if the user is the owner of the book
+  end
+
+  def destroy?
+    user == record.user # Allow deleting a book if the user is the owner of the book
+  end
```

Comment:
we can see that we use `user == record.user` several times, for make it more DRY, we can declare it as a method in `ApplicationPolicy`

```ruby
class ApplicationPolicy
  # ...

  protected

  def owner?
    user == record.user
  end
end

# in book_policiy

def update?
  owner?
end
```